### PR TITLE
test(ic-http-certification): add more tests to example projects

### PR DESCRIPTION
**Pull request by nathanosdev**
#395 must be merged before this one

### CODE REVIEW

**Review by przydatek — DISMISSED** [review]
LGTM, with some minor suggestions. Thanks!

File: examples/http-certification/assets/README.md [hunk 1]
```diff
@@ -224,15 +225,6 @@ fn certify_all_assets() {
...
         },
-        AssetConfig::Pattern {
-            pattern: "**/*.svg".to_string(),
-            content_type: Some("image/svg+xml".to_string()),
-            headers: get_asset_headers(vec![(
-                "cache-control".to_string(),
-                IMMUTABLE_ASSET_CACHE_CONTROL.to_string(),
-            )]),
-            encodings: vec![],
-        },
```
[the remark this answers is not in the document]
Comment (nathanosdev) [reply examples/http-certification/assets/README.md:235]: Yes exactly, I think the example _used_ to have an SVG asset but I removed it at some point and didn't remove the corresponding config.

File: examples/http-certification/assets/src/tests/src/http.spec.ts [hunk 2]
```diff
@@ -82,38 +90,278 @@ describe('Assets', () => {
...
+      headers: [],
+      body: new Uint8Array(),
+      certificate_version: [],
+    };
+
+    const response = await actor.http_request(request);
+
+    expect(response.status_code).toBe(301);
+    expectHeader(response.headers, ['location', '/']);
+    // enable when additional headers can be added to redirect responses
```
[the remark this answers is not in the document]
Comment (nathanosdev) [reply examples/http-certification/assets/src/tests/src/http.spec.ts:106]: I have it already addressed in the next PR

File: examples/http-certification/assets/src/tests/src/http.spec.ts [hunk 3]
```diff
@@ -82,38 +90,278 @@ describe('Assets', () => {
...
+      CERTIFICATE_VERSION,
+    );
+
+    expect(verificationResult.verificationVersion).toEqual(CERTIFICATE_VERSION);
+    expect(verificationResult.response?.statusCode).toBe(301);
+    expect(verificationResult.response?.headers).toContainEqual([
+      'location',
+      '/',
+    ]);
+    // enable when additional headers can be added to redirect responses
```
[the remark this answers is not in the document]
Comment (nathanosdev) [reply examples/http-certification/assets/src/tests/src/http.spec.ts:125]: I have it already addressed in the next PR

File: examples/http-certification/assets/README.md [hunk 4]
```diff
@@ -323,13 +327,41 @@ fn serve_metrics() -> HttpResponse<'static> {
...
+dfx start --background --clean
+```
+
+Then, deploy the canister:
+
+```shell
+dfx deploy http_certification_assets_backend
+```
+
+You can now access the canister's assets by navigating to the canister's URL in a web browser. The URL can also be found using the following command, making sure to replace `backend` with the name of the canister:
```
[the remark this answers is not in the document]
Comment (nathanosdev) [reply examples/http-certification/assets/README.md:350]: Thanks! This was an oversight

**Review by przydatek — APPROVED** [review]

**Pull request merged**